backend/src/main/java/com/team4/backend/model/InternshipContract.java

```diff
@@ -65,4 +65,30 @@ public InternshipContract(String id,
         this.monitorSignature = monitorSignature;
         this.internshipManagerSignature = internshipManagerSignature;
     }
+
```

Comment:
Just a remark but this 2 methods should be in a util class and should be tested(I know you are tired but it is what it is),
you can change it after or in the next sprint in the refactor